nextflow_schema.json

```diff
@@ -483,6 +489,116 @@
                 }
             }
         },
+        "locidex": {
+            "title": "Locidex",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "lx_min_evalue": {
+                    "type": "number",
+                    "default": 0.0001,
+                    "description": "Minimum e-value required for match.",
+                    "minimum": 1e-18,
+                    "maximum": 100
+                },
+                "lx_min_dna_len": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "Global minimum query length of DNA strand.",
+                    "minimum": 1
+                },
+                "lx_min_aa_len": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "Global minimum query length of an AminoAcid strand.",
+                    "minimum": 1
+                },
+                "lx_max_dna_len": {
+                    "type": "integer",
+                    "default": 10000000,
```

Comment:
How is this used? Since this is set to 10 million, would this mean that any sequence/contig > 10 million would not work in locidex?

Reply:
I currently do not quite know, I used the description within Locidex to add the description but I need to review this behaviour still.

Reply:
Okay, thanks. default of 10 million is likely fine since it encompasses the size of data we are running mikrokondo on and can always be adjusted at runtime.